### ISSUE TITLE
elm-generate app without package.json

### ISSUE THREAD

**Issue by hulufei** [rogeriochaves/spades#20]

**Comment by rogeriochaves** [comment]
Could you explain it better? I just tried and I can see the package.json there

```
» ls
elm.json                 package.json             package.static.json      server.js                src                      webpack.config.js        webpack.config.static.js
```

**Comment by hulufei** [comment]
@rogeriochaves try `elm-generate app MyProject --serverless`

**Comment by rogeriochaves** [comment]
It also worked for me. What's your OS, node and npm versions?

Also, could you try:
```
npm -g uninstall elm-on-spades
npm -g install git+https://github.com/rogeriochaves/spades.git
elm-generate app MyProject --serverless
```

**Comment by thetechnaddict** [comment]
Same problem solved.
- macOS 10.13.6
- node 10.6.0
- npm 6.4.1

Upgraded node to latest stable:

```
sudo npm cache clean -f
sudo npm install -g n
```
uninstalled / installed elm-on-spades as above

Now works as expected. :-)

**Comment by rogeriochaves** [comment]
Sorry for the long delay, it was actually a race condition issue, sometimes you got lucky and the project is generated correctly, some other times package.json was not there

I fixed it now on #25, please reopen this if any issues ;)
